content/en/blog/_posts/2023-10-18-gateway-api-time-to-switch-i2gw/index.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: blog
+title: "Time to Upgrade; Migrating to Gateway API"
```

Comment:
```suggestion
title: "Use ingress2gateway to Help Migration to the Gateway API"
```

Let's not imply that everyone has to upgrade their cluster. Sometimes we want to make that assertion; this is not that time.

Reply:
What about something like this? 

- "Introducing ingress2gateway, Simplifying Upgrades to Gateway API"

While I agree that we don't want to suggest that everyone needs to upgrade to Gateway API, I do think it's important to show that it is an upgrade.

Reply:
```suggestion
title: "Introducing ingress2gateway: Simplifying Upgrades to Adopt Gateway API
```

Reply:
I went with `Introducing ingress2gateway; Simplifying Upgrade to Gateway API`

Reply:
IMO you either need "the Upgrade" or "Upgrades", would personally prefer "Upgrades" as it is more generic and would not be confused with a requirement to upgrade to Gateway API.